client/context/EditorContext.tsx

```diff
@@ -1,7 +1,13 @@
+import { Snackbar } from "@mui/material";
 import React, { createContext, useContext, useReducer } from "react";
 import { editObjectCatalogUrl, getObjectModelsDatastreamsUrl } from "../util/routes";
 import { useFetchContext } from "./FetchContext";
 
+interface SnackbarState {
```

Comment:
Is this the right place to define this? Is it used anywhere?

Reply:
I think this thread may have gotten hidden by GitHub's collapsed conversations feature -- I believe the question remains open.